packages/manager/apps/dedicated/client/app/ip/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -272,8 +313,20 @@ angular.module('Module.ip.controllers').controller(
     }
 
     redirectToPaymentPage() {
-      const productToOrder = this.IpAgoraOrder.constructor.createProductToOrder(
-        {
+      let productToOrder = null;
+      if (this.isPrivateCloudOffer) {
+        productToOrder = this.IpAgoraOrder.constructor.createProductToOrder({
+          productId: 'privateCloud',
+          duration: get(this.model.params, 'selectedOffer.duration'),
+          pricingMode: get(this.model.params, 'selectedOffer.pricingMode'),
+          country: get(this.model.params, 'selectedCountry.code'),
+          destination: get(this.model, 'selectedService.serviceName'),
+          serviceName: get(this.model, 'selectedService.serviceName'),
```

Comment:
destination and serviceName has the same value

Reply:
yes those are agora parameters passed to the order funnel, i need to check if i can remove the destination but not sure about that